README.md

```diff
@@ -112,6 +112,24 @@ Then run start the server with:
 
 It will automatically startup on reboot.
 
+Exporting/Importing Default collections
+===========================
+
+### Exporting
+Follow the below steps to successfully export and import the default collections ( Users, Channels, Clients, ContactGroups, Mediators )
```

Comment:
Rather than saying default collections, let's clarify the task a bit: maybe say something like "Follow the below steps to export and import the server metadata configuration ... By default the Users, Channels, Clients, ContactGroups and Mediators collections will be exported"